playbook/app/pb_kits/playbook/pb_tooltip/docs/example.yml

```diff
@@ -4,6 +4,8 @@ examples:
   - tooltip_default: Default
   - tooltip_interaction: Content Interaction
   - tooltip_selectors: Using Common Selectors
+  - tooltip_margin: Margin
```

Comment:
I think this was left in by mistake, its adding an empty margin doc on the rails side!